docs/enhancements/ove-ignition-generation.md

```diff
@@ -0,0 +1,165 @@
+---
+title: ove-ignition-generation
+authors:
+  - "@ovishlit"
+creation-date: 2025-07-15
+last-updated: 2025-07-15
+---
+
+# OVE Ignition Generation
+
+## Summary
+
+Enable serving OpenShift Virtualization Edition (OVE) ISOs with custom ignition containing user-provided values (SSH keys, pull secrets). OVE ISOs are 40GB+ self-contained images that include the entire OpenShift release payload, operators, and custom images for a specific version, designed for disconnected environment deployments. 
```

Comment:
Are we going to continue using the OVE name for this?

From assisted, we don't care what is pre-loaded into this ISO right? If someone created an image with other content preloaded into it, would what we build here work the same way? If so I think we should use a more generic name.